packages/dialtone-vue2/recipes/conversation_view/emoji_row/emoji_row.vue

```diff
@@ -106,52 +106,53 @@ export default {
   }
 
   &__reaction {
-  --emoji-item-color-inset-shadow: transparent;
-  --emoji-item-color-foreground: var(--dt-action-color-foreground-muted-default);
-  --emoji-item-color-background: var(--dt-action-color-background-muted-hover);
-
-  padding: var(--dt-space-300) var(--dt-space-400);
-  border-radius: var(--dt-size-radius-pill);
-  border: 0;
-  color: var(--emoji-item-color-foreground);
-  background-color: var(--emoji-item-color-background);
-  box-shadow: inset 0 0 0 var(--dt-size-border-150) var(--emoji-item-color-inset-shadow);
-
-  &.dt-emoji-row__picker {
-    padding: var(--dt-space-200) var(--dt-space-350);
-  }
+    --emoji-item-color-inset-shadow: transparent;
+    --emoji-item-color-foreground: var(--dt-action-color-foreground-muted-default);
+    --emoji-item-color-background: var(--dt-action-color-background-muted-hover);
+
+    padding: var(--dt-space-300) var(--dt-space-400);
+    border-radius: var(--dt-size-radius-pill);
+    border: 0;
+    color: var(--emoji-item-color-foreground);
+    background-color: var(--emoji-item-color-background);
+    box-shadow: inset 0 0 0 var(--dt-size-border-150) var(--emoji-item-color-inset-shadow);
+    height: var(--dt-size-550);
```

Comment:
This is the only real change. Everything else is just correcting the indentation.